ranker/modules/omnicorp_overlay.py

```diff
@@ -18,7 +18,7 @@
 logger = logging.getLogger(__name__)
 
 CACHE_HOST = os.environ.get("CACHE_HOST", "localhost")
-CACHE_PORT = os.environ.get("CACHE_PORT", "6379")
+CACHE_PORT = os.environ.get("CACHE_PORT", "6380")
```

Comment:
It's just that the omnicorp cache is running behind that port.

Reply:
its just an old cruft from robokop components running on same machine. We had it use that same ports since. But we could change it to standard port here, since we will set that port in the chart via env var